### PR TITLE
New Airbyte connectors docs

### PR DESCRIPTION
**Pull request by jman0119**
Creating docs for the 15 new Airbyte connectors we recently added.

- - -
This change is [Reviewable](https://reviewable.io/reviews/estuary/flow/1024)

### CODE REVIEW

File: site/docs/reference/Connectors/capture-connectors/chargebee.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+### Properties
+
+#### Endpoint
+
+| Property | Title | Description | Type | Required/Default |
+|---|---|---|---|---|
+| `/site_api_key` | API Key | Chargebee API Key. | string | Required |
+| `/site` | Site | The site prefix for your Chargebee instance. | string | Required |
+| `/start_date` | Start Date | UTC date and time in the format 2021-01-25T00:00:00Z. Any data before this date will not be replicated. | string | Required |
+| `/product_catalog` | Product Catalog | Product Catalog version of your Chargebee site. Instructions on how to find your version you may find under 'API Version' section here: https://apidocs.chargebee.com/docs/api?prod_cat_ver=2 | string | Required |
```
Comment (oliviamiannone) [inline site/docs/reference/Connectors/capture-connectors/chargebee.md:48]: If possible, it'd be great to not expose the URL as the link itself. Also I think the doc link is wrong. 
ie format like: [in the Chargebee docs](https://apidocs.chargebee.com/docs/api/versioning?prod_cat_ver=2)
(That's the correct link there btw!)

File: site/docs/reference/Connectors/capture-connectors/mixpanel.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+You configure connectors either in the Flow web app, or by directly editing the catalog specification file.
+See [connectors](../../../concepts/connectors.md#using-connectors) to learn more about using connectors. The values and specification sample below provide configuration details specific to the MixPanel source connector.
+
+### Properties
+
+#### Endpoint
+
+| Property | Title | Description | Type | Required/Default |
+|---|---|---|---|---|
+| `/project_id` | Project ID | Your project ID number. See the docs for more information on how to obtain this: https://help.mixpanel.com/hc/en-us/articles/115004490503-Project-Settings#project-id | integer | Required |
```
Comment (oliviamiannone) [inline site/docs/reference/Connectors/capture-connectors/mixpanel.md:43]: Same comment about URLs here. I know this comes from the connector spec but it looks good to reformat the link as text for the docs!

File: site/docs/reference/Connectors/capture-connectors/sendgrid.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+* [Blocks](https://docs.sendgrid.com/api-reference/blocks-api/retrieve-all-blocks)
+* [Bounces](https://docs.sendgrid.com/api-reference/bounces-api/retrieve-all-bounces)
+* [Invalid emails](https://docs.sendgrid.com/api-reference/invalid-e-mails-api/retrieve-all-invalid-emails)
+* [Spam reports](https://docs.sendgrid.com/api-reference/spam-reports-api/retrieve-all-spam-reports)
+
+By default, each resource is mapped to a Flow collection through a separate binding.
+
+## Prerequisites
+
+* SendGrid [API Key] (https://docs.sendgrid.com/api-reference/api-keys/create-api-keys) for authentication.
```
Comment (oliviamiannone) [inline site/docs/reference/Connectors/capture-connectors/sendgrid.md:37]: There's an empty space between the two brackets causing this to not render

File: site/docs/reference/Connectors/capture-connectors/zendesk-chat.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+* [Skills](https://developer.zendesk.com/rest_api/docs/chat/skills#list-skills)
+* [Roles](https://developer.zendesk.com/rest_api/docs/chat/roles#list-roles)
+* [Routing Settings](https://developer.zendesk.com/rest_api/docs/chat/routing_settings#show-account-routing-settings)
+
+By default, each resource is mapped to a Flow collection through a separate binding.
+
+## Prerequisites
+
+* A Zendesk Account with permission to access data from accounts you want to sync.
+* An Access Token (https://developer.zendesk.com/rest_api/docs/chat/auth). We recommend creating a restricted, read-only key specifically for Estuary access to allow you to control which resources Estuary should be able to access.
```
Comment (oliviamiannone) [inline site/docs/reference/Connectors/capture-connectors/zendesk-chat.md:33]: no square brackets so this isn't rendering

**Review by oliviamiannone — COMMENTED** [review]
Hey Jenny, thanks so much for putting all these together! There are a couple link issues I called out. apart from that, the main thing is these connectors should all be linked from the main connectors landing page - https://docs.estuary.dev/reference/Connectors/capture-connectors/

**Comment by jman0119** [comment]
Thanks @oliviamiannone! I've made the changes - ready for review again, thanks!

**Review by oliviamiannone — APPROVED** [review]
Thanks Jenny!

**Pull request merged**